cloud/ec2_client.go

```diff
@@ -973,7 +973,9 @@ func (c *awsClientImpl) GetVolumeIDs(ctx context.Context, h *host.Host) ([]strin
 		if err != nil {
 			return nil, errors.Wrap(err, "error getting devices")
 		}
-		h.Volumes = makeVolumeAttachments(devices)
+		if err := h.SetVolumes(makeVolumeAttachments(devices)); err != nil {
```

Comment:
The only issue I see with this is that other operations rely on Size and CreatedBy to be set if the volume was created by a user, which makeVolumeAttachments doesn't handle. CreatedBy can be populated from the host document if applicable, but I'm not totally sure about size.

Reply:
I'm not sure I understand what this means because I have no idea what the distinction is between the volume collection and volume attachments/size stored in the host document.

I only did this because `TestOnUp` asserts that the database saved the volumes to the host document so I'm assuming it's supposed to be stored on the host. I changed this to also include the volume size when this value is being initialized.

I don't know if we actually want to save the host volume attachments or not here. If `OnUp` is _not_ supposed to save the volume attachments/total size, then can we just get rid of the assertions from `TestOnUp`?